tpl/collections/collections.go

```diff
@@ -587,3 +587,48 @@ func (ns *Namespace) Union(l1, l2 interface{}) (interface{}, error) {
 		return nil, errors.New("can't iterate over " + reflect.ValueOf(l1).Type().String())
 	}
 }
+
+// Uniq takes in a slice or array and returns a slice with subsequent
+// duplicate elements removed.
+func (ns *Namespace) Uniq(l interface{}) (interface{}, error) {
+	if l == nil {
+		return make([]interface{}, 0), nil
+	}
+
+	lv := reflect.ValueOf(l)
+	lv, isNil := indirect(lv)
+	if isNil {
+		return nil, errors.New("Can't uniquify a nil value")
```

Comment:
"uniquify" doesn't exist.  Change this to simply `invalid nil argument to Uniq`.